assets/js/nps.js

```diff
@@ -0,0 +1,41 @@
+function fNpsApi() {
+    // var searchTerm = document.querySelector('#searchTerm').value;
+
+    var searchTerm = 'CA';
+    var sNpsApi = 'https://developer.nps.gov/api/v1/campgrounds?' +
+        'stateCode=' +
+        searchTerm +
+        '&limit=500' +
+        '&api_key=nNVtVCtPxYIdqgXcnAhfGjtCq9cW1SUGJ6AnV68u'
+    // Create a variable to hold the value of rating
+    // var rating = document.querySelector('#rating').value;
+    //  https://developer.nps.gov/api/v1/parks?parkCode=acad&api_key=nNVtVCtPxYIdqgXcnAhfGjtCq9cW1SUGJ6AnV68u
+    fetch(sNpsApi)
+        .then(function (response) {
+            return response.json();
+        })
+        .then(function (response) {
+            console.log(response);
+            aNPS[0].markerName = response.data[0].name;
```

Comment:
I noticed while looking through the API that name describes a lot of different data points. I havent run this code yet so forgive me if I am wrong but it looked like it might pull up the park name, activities, topics, designations for memorials, designations for updated hours, etc. When I prompted a search for 'name' after pulling up Ohio national parks I got 300+ results. I noticed that for the actual park names they're designated as 'fullName' this only gave me 14 results. Let's test it out today in the meeting!